cakebuild.cmd

```diff
@@ -0,0 +1,2 @@
+@echo off
+powershell ./build.ps1 %*
```

Comment:
Not on my machine. That's why I wrote it.

BTW, I should have clarified in the description of the PR that cakebuild.cmd and cake.cmd are placeholders. After we remove the old NUnit.proj and build scripts, I'll rename some of these. This one will be just build.cmd and cake.cmd may not be needed.

BTW#2: if you run this on a 32-bit windows machine, you should specify -experimental to cake or -Experimental to build.ps1. If you use -mono, there is a compiler error that shows up, which I will report to the cake guys.